src/dataflow/lib/iot.ts

```diff
@@ -1,12 +1,32 @@
 import { MqttClient, connect } from "mqtt";
 import { IStores } from "../models/stores/dataflow-stores";
 import { getSignedUrl } from "../utilities/aws";
+import { HubChannelModel } from "../models/stores/hub-store";
 import * as AWS from "aws-sdk";
 
 const AWS_REGION = "us-east-1";
 const COGNITO_POOL = "us-east-1:153d6337-3421-4c34-a21f-d1d2143a5091";
 const AWS_IOT_ENDPOINT_HOST = "a2zxjwmcl3eyqd-ats.iot.us-east-1.amazonaws.com";
-const MQTT_TOPIC = "test_topic";
+// WTD This is for testing purposes only. Remove when we add users and hub owners.
+const OWNER_ID = "123";
+// WTD This is for testing purposes only. Sensor send interval should be set by user.
+const SEND_INTERVAL = 2;
+
+export enum TopicType {
+  hubChannelInfo = "hubChannelInfo",
+  hubSensorValues = "hubSensorValues",
+  hubStatus = "hubStatus",
+  hubCommand = "hubCommand",
+  hubRelays = "hubRelays"
+}
+
+export const TopicInfo = {
+  [TopicType.hubChannelInfo]: { topicSuffix: "/devices" },
+  [TopicType.hubSensorValues]: { topicSuffix: "/sensors" },
+  [TopicType.hubStatus]: { topicSuffix: "/status" },
+  [TopicType.hubCommand]: { topicSuffix: "/command" },
+  [TopicType.hubRelays]: { topicSuffix: "/actuators" }
+};
```

Comment:
As mentioned below, I'd be inclined to have the `/` be part of the topic structure below rather than being included here. I wish there were a better name than `topicSuffix` -- given that in MQTT they're called levels, would `leafLevel` be any better? It seems like there ought to be a standard nomenclature for referring to the last level.

Reply:
I'm ok with `leafLevel`.  I looked through the MQTT spec a little (level is definitely right), but couldn't find a name for that final level.  `leafLevel` makes sense to me.  Fixed.